overviewer_core/textures.py

```diff
@@ -1995,7 +2008,7 @@ def stairs(self, blockid, data):
     # sandstone, red sandstone, and quartz stairs have special top texture
     if blockid == 128:
```

Comment:
Consider replacing this entire special case if block with a dictionary:

```python
special_tops = {
    128: "assets/minecraft/textures/block/sandstone_top.png",
    156: "assets/minecraft/textures/block/quartz_block_top.png",
    180: "assets/minecraft/textures/block/red_sandstone_top.png",
    11375: "assets/minecraft/textures/block/quartz_block_top.png",
}

if blockid in special_tops:
    texture = self.load_image_texture(special_tops[blockid]).copy()
```

This scales better if more blocks need to be added.